src/v2/view-builder/views/duo/ChallengeDuoAuthenticatorView.js

```diff
@@ -0,0 +1,35 @@
+import { loc } from 'okta';
+import Duo from 'duo';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const Body = BaseForm.extend({
+
+  className: 'duo-authenticator-challenge',
+  noButtonBar: true,
+
+  title  () {
+    return loc('oie.duo.verify.title', 'login');
+  },
+
+  postRender: function () {
+    const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    // TODO Check contextualData.integrationType once we support other types than iframe
```

Comment:
Since these are returned via our API - should we add basic checks to ensure the parsed values `host` and `signedToken` exist before passing them into the Duo initializer?

Reply:
There is no jira for TODO, it is future work that will happen at some point.

Passing wrong data to the Duo.init will just fail that iframe from loading. I think we can rely on them checking that the values are valid, otherwise here we would just check they are not undefined or empty, but we can delegate that to duo

Reply:
Looking at this again, I'll add some basic error handling for the Duo.init call @jmelberg-okta